mock_ros2_project/action_server.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+"""
+ROS2 Action Server Example
+
+This node demonstrates how to create an action server that provides
+a Fibonacci sequence generation action.
+"""
+
+import time
+from typing import Optional
+
+import rclpy
+from rclpy.action import ActionServer, GoalResponse
```

Comment:
`GoalResponse` is not referenced anywhere in the file. Unless you plan to add a goal callback that uses it, the symbol can be dropped from the import list.

```suggestion
from rclpy.action import ActionServer
```